service/history/queue/interface.go

```diff
@@ -38,50 +38,56 @@ type (
 		ReverseMatch bool
 	}
 
-	// JobInfo indicates the scope of a Job and its progress
-	JobInfo interface {
-		QueueID() int
+	// ProcessingQueueStates is a list of ProcessingQueueState
+	ProcessingQueueStates []ProcessingQueueState
```

Comment:
Now that we have` Queues() ProcessingQueues` do we still need this alias?

Reply:
Removed. I don't think they are necessary, but I thought you want me to add those alias?

Reply:
Sorry for the confusion. I wanted to say it's good to have alias if we use It :) Thanks!